src/loc/en_US.json

```diff
@@ -80,7 +116,35 @@
 
         },
         "styleTemplate": {
-
+            "fillColor": "Fill",
+            "backgroundFillColor": "Background fill",
+            "opacity": "Opacity",
+            "strokeWidth": "Line width",
+            "textSize": "Text size",
+            "pointSize": "Point size",
+            "showLengthArea": "Show length/area",
+            "showSegments": "Show segments",
+            "stroke": "Line",
+            "strokeTypeSolid": "Solid line",
+            "strokeTypeDotted": "Dotted line",
+            "strokeTypeDashed" : "Dashed line",
+            "strokeTypeDashedDotted" : "Dash-dotted line",
+            "dimensions": "Dimensions",
+            "margins": "Margins",
+            "rotation": "Rotation",
+            "degreesRotation": "Degrees",
+            "marginForText": "Margin",
+            "frame": "Frame",
+            "margin": "Margin",
+            "point": "Point",
+            "pointTypePoint": "Point",
+            "pointTypeCircle": "Circle",
+            "pointTypeX": "X",
+            "pointTypePlus": "Cross",
```

Comment:
"pointTypePlus": "Cross" should be "pointTypePlus": "Plus"

Reply:
"Kors" works in swedish and "Plus" would also be appropriate in swedish, so I don't know which is best.